src/components/Articles.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import styled from 'styled-components';
+import Readme1 from '../images/readme1.jpg';
+import Readme2 from '../images/readme2.jpg';
+
+export const article = [
+  { header: 'Guide for beginners', paragraf: 'Join our growing community and let us support your well-being. If you are a beginner and do not dare to try yoga because you think you are too stiff and immobile for some of the positions, then you have come to the right place!' },
+  { header: 'Yoga retreat ', paragraf: 'This is not just a yoga retreat, meditation course or wellness trip – it is a complete experience for body and soul. Our concept is designed to, togetherwith yoga and meditation and other exciting spiritual tools, allow you to explore movement, emotions, stillness and above all – yourself.' }
+]
```

Comment:
This was super smart, making the articles in to an array. It makes the Article-function so much easier to read!